app/routes/__index/certificate/index.tsx

```diff
@@ -1,11 +1,65 @@
-import { Heading } from '@chakra-ui/react';
+import { Flex, Center, Button, Divider } from '@chakra-ui/react';
+import type { Certificate } from '@prisma/client';
+import { useState } from 'react';
+
+import Loading from '~/components/certificate/loading';
+import CertificateDisplay from '~/components/certificate/certificate-display';
+import Request from '~/components/certificate/request';
+import Description from '~/components/certificate/description';
 
 export default function CertificateIndexRoute() {
-  return (
-    <div>
-      <Heading as="h1" size="3xl">
-        Certificate Info or option to request a certificate
-      </Heading>
-    </div>
-  );
+  const [request, setRequest] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  function handleSubmit() {
+    setLoading(true);
+    setTimeout(() => {
+      setRequest(true);
+      setLoading(false);
+    }, 5000);
+  }
+
+  let curDate: Date = new Date(2023, 2);
```

Comment:
value is immutable, so better make it `const` + I think it is better to avoid to main shortcuts is variable naming, just to make it more clear, so `currentDate`

Reply:
I miss that lul, and I agree with the naming it's much better your way